va/hci/nodeset/kustomization.yaml

```diff
@@ -0,0 +1,28 @@
+---
+apiVersion: kustomize.config.k8s.io/v1alpha1
+kind: Component
+
+patches:
+  - target:
+      kind: OpenStackDataPlaneNodeSet
+    patch: |-
+      - op: add
+        path: /spec/nodeTemplate/networks/-
+        value:
+          name: swift
+          subnetName: subnet1
+      - op: add
+        path: /spec/nodes/edpm-compute-0/networks/-
```

Comment:
This is failing because the name will always have a random string appended to it:
```
       nodes:
            edpm-compute-eeua7yef-0:
                ansible:
                    host: 192.168.122.100
                hostName: compute-eeua7yef-0
                networks:
```
I get this error:
```
add operation does not apply: doc is missing path: \\\"/spec/nodes/edpm-compute-0/networks/-\\\": missing value\""'
```

Reply:
Because of the dependency on  https://github.com/openstack-k8s-operators/ci-framework/pull/2301 the ci-framework when the job is run by the ci-framework I already end up having those values. Thus, I could simply remove those lines.
```
        networks:
        -   defaultRoute: true
            name: ctlplane
            subnetName: subnet1
        -   name: internalapi
            subnetName: subnet1
        -   name: storage
            subnetName: subnet1
        -   name: tenant
            subnetName: subnet1
        nodes:
            edpm-compute-eeua7yef-0:
                ansible:
                    host: 192.168.122.100
                hostName: compute-eeua7yef-0
                networks:
                -   defaultRoute: true
                    fixedIP: 192.168.122.100
                    name: ctlplane
                    subnetName: subnet1
                -   name: internalapi
                    subnetName: subnet1
                -   name: storage
                    subnetName: subnet1
                -   name: storagemgmt
                    subnetName: subnet1
                -   name: swift
                    subnetName: subnet1
                -   name: tenant
                    subnetName: subnet1
            edpm-compute-eeua7yef-1:
                ansible:
                    host: 192.168.122.101
                hostName: compute-eeua7yef-1
                networks:
                -   defaultRoute: true
                    fixedIP: 192.168.122.101
                    name: ctlplane
                    subnetName: subnet1
                -   name: internalapi
                    subnetName: subnet1
                -   name: storage
                    subnetName: subnet1
                -   name: storagemgmt
                    subnetName: subnet1
                -   name: swift
                    subnetName: subnet1
                -   name: tenant
                    subnetName: subnet1
            edpm-compute-eeua7yef-2:
                ansible:
                    host: 192.168.122.102
                hostName: compute-eeua7yef-2
                networks:
                -   defaultRoute: true
                    fixedIP: 192.168.122.102
                    name: ctlplane
                    subnetName: subnet1
                -   name: internalapi
                    subnetName: subnet1
                -   name: storage
                    subnetName: subnet1
                -   name: storagemgmt
                    subnetName: subnet1
                -   name: swift
                    subnetName: subnet1
                -   name: tenant
                    subnetName: subnet1
```